src/Components/signin/signin-header/index.js

```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import styled from 'styled-components';
+import React from './node_modules/react';
+import styled from './node_modules/styled-components';
```

Comment:
Remove `./node_modules/` since a folder called `node_modules` does not exist in the SigninHeader folder. Or use full path to root `node_modules`.